pkg/cli/setup/application_test.go

```diff
@@ -33,6 +33,7 @@ func Test_ScaffoldApplication_CreatesBothFiles(t *testing.T) {
 
 	require.FileExists(t, filepath.Join(directory, ".rad", "rad.yaml"))
 	require.FileExists(t, filepath.Join(directory, "app.bicep"))
+	require.FileExists(t, filepath.Join(directory, "bicepconfig.json"))
```

Comment:
Should we have a test that validates the content of `bicepconfig.json`? I know it's hardcoded right now, but I thinking about future us ⌛

Reply:
This should check that the generated file vs template are the same: https://github.com/radius-project/radius/blob/e2eae38d5d62429d371c604b9426cc3e81b83617/pkg/cli/setup/application_test.go#L58

Let me know if you meant something different

Reply:
This is fine for now 👍 It will get more complicated in the future, which is why I wanted some kind of test to exist.